### ISSUE TITLE
Turn back some old locations, that was abandoned

### ISSUE THREAD

**Issue by GuardianDll** [CleverRaven/Cataclysm-DDA#63016]
### Is your feature request related to a problem? Please describe.

#55894 removed some old mods, that have had lack of maintaining. Some of them were merged into the game afterwards, but some are still here, forgotten and abandoned

### Solution you would like.

Polish and add them into main game:

Medium level:
<details>
  <summary> - [ ] Urban_Development/building_jsons/urban_19_victorian.json </summary>

![image](https://user-images.githubusercontent.com/67688115/211005221-43441cf3-cc3a-4b4f-9cb1-2304b816743d.png)
![image](https://user-images.githubusercontent.com/67688115/211005262-a9a2e4c9-20b0-45e1-bbde-df979e6c6b7b.png)
![image](https://user-images.githubusercontent.com/67688115/211005292-ab138988-934b-4a3e-80b9-7cc2bbcf51e3.png)
![image](https://user-images.githubusercontent.com/67688115/211005314-63379636-02c4-4729-a90c-29cdc2c234db.png)
![image](https://user-images.githubusercontent.com/67688115/211005339-10a3aa52-6f04-4dad-bfc4-2e0f4202690a.png)
![image](https://user-images.githubusercontent.com/67688115/211005361-89d530eb-a4b8-4c15-b4a8-b33305bcb0f0.png)

</details>

<details>
  <summary> - [ ] Urban_Development/building_jsons/urban_20_duplex.json </summary>

![image](https://user-images.githubusercontent.com/67688115/211005880-aae314f9-3762-4997-9111-9d39a3b3d479.png)
![image](https://user-images.githubusercontent.com/67688115/211005913-7b14832c-d62f-48df-950e-7464de4b1bbe.png)
![image](https://user-images.githubusercontent.com/67688115/211005944-a09e52fa-6b2f-4253-a294-31ca3ed10bf5.png)
![image](https://user-images.githubusercontent.com/67688115/211005978-fe1c879d-e2eb-4808-a112-a14d446045f3.png)

</details>

<details>
  <summary> - [ ] Urban_Development/building_jsons/urban_21_house </summary>

![image](https://user-images.githubusercontent.com/67688115/211006130-f52a9837-0dfc-4c92-8da5-5a68384ce3f0.png)
![image](https://user-images.githubusercontent.com/67688115/211006185-74bd10cd-e3ee-445e-8354-5df563dbdf8a.png)

</details>

<details>
  <summary> - [ ] ididididididid </summary>

</details>

<details>
  <summary> - [ ] ididididididid </summary>

</details>

<details>
  <summary> - [ ] ididididididid </summary>

</details>

<details>
  <summary> - [ ] ididididididid </summary>

</details>

<details>
  <summary> - [ ] ididididididid </summary>

</details>

### Describe alternatives you have considered.

<details>
  <summary> - [ ] ididididididid </summary>

</details>

**Comment by LovamKicsiGazsii** [comment]
Yeah, lotta work needed for these to be mainlined. Curstwist has done a great job at restoring old urbdev places - merging maps, migrating to dedicated palettes, adding roofs, new furniture, and SUS item groups... these locations should be given the same treatment as well.

**Comment by Maleclypse** [comment]
Did a crater spawn on the projects building or does it naturally spawn looking all exploded?

**Comment by GuardianDll** [comment]
> Did a crater spawn on the projects building or does it naturally spawn looking all exploded?

latter

**Comment by LovamKicsiGazsii** [comment]
Are you planning to work on this project yourself or would you like some help with it?

**Comment by GuardianDll** [comment]
I plan to do it in some future, but currently I'm busy with #62915 and Xedra Evolved mod stuff

**Comment by kevingranade** [comment]
Wishlist issue, closing.

**Comment by GuardianDll** [comment]
What in
> I plan to do it in some future, but currently I'm busy

makes you think its wishlist issue?

**Comment by anothersimulacrum** [comment]
https://github.com/CleverRaven/Cataclysm-DDA/issues/58865#issuecomment-1475384889
